package.json

```diff
@@ -205,7 +205,6 @@
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/plugin-proposal-private-methods": "^7.18.6",
-    "@babel/plugin-syntax-import-assertions": "^7.23.3",
```

Comment:
can you also remove `core-js` in this PR

Reply:
yup forgot about that